pytensor/tensor/optimize.py

```diff
@@ -560,7 +560,10 @@ def L_op(self, inputs, outputs, output_grads):
         implicit_f = grad(inner_fx, inner_x)
 
         df_dx, *df_dtheta_columns = jacobian(
-            implicit_f, [inner_x, *inner_args], disconnected_inputs="ignore"
+            implicit_f,
+            [inner_x, *inner_args],
+            disconnected_inputs="ignore",
+            vectorize=True,
```

Comment:
This triggers https://github.com/pymc-devs/pytensor/issues/1550 in the existing example (as seen in https://github.com/pymc-devs/pytensor/pull/1558)